File: src/main/java/com/mbsl/controller/ApprovelProcessController.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package com.mbsl.controller;

import com.mbsl.model.ApprovelProcess;
import com.mbsl.service.ApprovelProcessService;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.http.HttpStatus;
import org.springframework.http.ResponseEntity;
import org.springframework.web.bind.annotation.RequestBody;
import org.springframework.web.bind.annotation.RestController;

/**
 *
 * @author MBSL2523
 */
@RestController
public class ApprovelProcessController implements IApprovelProcessController{

@Autowired
    private ApprovelProcessService approvelProcessService;


@Override
    public ResponseEntity<String> approveConfirm(@RequestBody ApprovelProcess approvelProcess) {

        return new ResponseEntity<>(approvelProcessService.approveConfirm(approvelProcess), HttpStatus.OK);
//          return new ResponseEntity<>(userTransfersService.userTransfers(user), HttpStatus.OK);
    }
    
}

File: src/main/resources/static/js/app/approvelProcessController.js
/* 
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/JSP_Servlet/JavaScript.js to edit this template
 */

app.controller("approvelProcessController", function ($scope, $http, $mdDialog, $filter) {





    $scope.approvelProcess = {

        test: "",

    };

    $scope.user.toDay = $filter('date')(new Date(), 'MM/dd/yyyy');


    $scope.approveConfirm = function () {

        var method = "POST";
        var url = 'appProcess';
        $scope.createUserBusy = $http({
            method: method,
            url: url,
            data: angular.toJson($scope.approvelProcess),
            headers: {
                'Content-Type': 'application/json'
            },
            transformResponse: [
                function (data) {
                    location.reload();
                }
            ]
        });
    };




});

